pandas/tests/io/test_html.py

```diff
@@ -869,6 +881,18 @@ def test_computer_sales_page(self):
         data = os.path.join(DATA_PATH, 'computer_sales_page.html')
         self.read_html(data, header=[0, 1])
 
+    def test_multiple_header_rows(self):
+        # Issue #13434
+        expected_df = DataFrame(data=[("Hillary", 68, "D"),
```

Comment:
whoosh we should never actually duplicate the test code (not sure if this is existing)

rather use a base class (or parametization)

if this is the current style can u pls create an issue to fix this (if not pls fix this!)